d2core/d2render/ebiten/ebiten_renderer.go

```diff
@@ -31,9 +31,9 @@ func CreateRenderer() (*Renderer, error) {
 
 	config := d2config.Get()
 
-	ebiten.SetCursorVisible(false)
+	ebiten.SetCursorMode(ebiten.CursorModeHidden)
```

Comment:
Good that you replaced depricated functions!

Reply:
I wonder if that fixes the issue of the cursor not hiding when toggling fullscreen on OSX..

Reply:
> I wonder if that fixes the issue of the cursor not hiding when toggling fullscreen on OSX..

Not likely. The deprecated functions of ebiten still invoke the new functions underneath.

```
$ go doc -src ebiten.SetCursorVisible

package ebiten // import "github.com/hajimehoshi/ebiten"

// SetCursorVisible is deprecated as of 1.11.0-alpha. Use SetCursorMode instead.
func SetCursorVisible(visible bool) {
	if visible {
		SetCursorMode(CursorModeVisible)
	} else {
		SetCursorMode(CursorModeHidden)
	}
}
```